modules/ROOT/pages/zero-to-hero.adoc

```diff
@@ -0,0 +1,293 @@
+= Zero to Hero
+
+The Enterprise Contract can be used to verify an image that has already been signed and attested by
+a build system. This document assumes you do not have such an image. It walks you through the
+process of signing and attestating an image, then how to use EC to validate those operations.
+
+== Required Tooling
+
+Before starting, the following tooling is required:
+
+* https://github.com/sigstore/cosign#installation[cosign]
+* https://enterprisecontract.dev/docs/user-guide/main/cli.html[ec cli]
+* https://github.com/containers/skopeo[skopeo] or
+  https://github.com/google/go-containerregistry/tree/main/cmd/crane[crane]
+* https://docs.docker.com/engine/reference/run/[docker] or https://podman.io/[podman] (optional)
+
+== Container Repository
+
+During this guide, image signatures and attestations will be pushed to the container repository
+where the image resides. Using a local registry is ideal for testing purposes. If you have a
+container runtime installed, e.g. https://docs.docker.com/engine/reference/run/[docker] or
+https://podman.io/[podman], you can start one with ease:
+
+[.console-input]
+[source, bash]
+----
+docker run -it -d -p 5000:5000 --restart=always --name registry registry:2
+# or with podman:
+# podman run -it -d -p 5000:5000 --restart=always --name registry registry:2
+----
+
+Alternatively, if you have access to a repository in an existing registry, e.g. quay.io or
+docker.io, then you can just use that.
+
+Set the environment variable `REPOSITORY` accordingly:
```

Comment:
Remove "environment" or add `export` to `export REPOSITORY`

Reply:
Why?

`FOO=BAR` is perfectly valid way of setting an environment variable. Without `export`, it just means that sub-processes won't get it, just the shell which is exactly what we want.

Reply:
I didn't think it was technically an **environment** variable if you didn't export.

Reply:
I'm not sure why github left the above comment twice.

Reply:
Actually, you're right. I think the right term here is a shell variable. I'll change it to just say variable as you suggested. Thank you for catching this. I learned something new.

Reply:
Went with "shell variable".